### PR TITLE
Make the text area resizable

### LINKED ISSUE #104: Note textarea should be resizable.
It's not possible to make the note textarea higher (i.e. display more lines) when adding a new note and it should be.

### PR DESCRIPTION
**Pull request by wirednkod**
Fixes #104

### CODE REVIEW

File: src/components/NoteManager/NoteManager.css [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-  flex-direction: column;
 }
 .note-manager textarea {
   flex: 1;
   padding: 1rem;
+  resize: vertical;
+}
+
+.note-manager .new-note button {
+  height: 5rem;
```
Comment (skoszuta) [inline src/components/NoteManager/NoteManager.css:11]: why move the add button to the right?

Comment (wirednkod) [reply src/components/NoteManager/NoteManager.css:11]: IMO - It fits better to the "delete" section (has also the button on the right) and also allows in an easy way the text area to be resizable without the button being moved with it. 
I could rework it for it to be under the text area - but movable buttons is not always the best way to go
🤷🏽

Comment (skoszuta) [reply src/components/NoteManager/NoteManager.css:11]: the button is squeezed to the right hand side in an awkward way and looks even more awkward once you increase the height of the textarea. the button moving lower as you resize a textarea is not an issue as you can see in the exact form you just used to post a comment here on github.

Comment (wirednkod) [reply src/components/NoteManager/NoteManager.css:11]: Will adjust as needed then and push on top of this PR

**Review by skoszuta — APPROVED** [review]
looks good, thank you 🙌

**Pull request merged**